pkg/eks/apiv2.go

```diff
@@ -87,7 +82,6 @@ func makeEndpointResolverFunc() aws.EndpointResolverWithOptionsFunc {
 		ec2.ServiceID:                    "AWS_EC2_ENDPOINT",
 		elasticloadbalancing.ServiceID:   "AWS_ELB_ENDPOINT",
 		elasticloadbalancingv2.ServiceID: "AWS_ELBV2_ENDPOINT",
-		sts.ServiceID:                    "AWS_STS_ENDPOINT",
```

Comment:
why don't we want this override?

Reply:
Errr.... Ops, I might have missed that. Let me take a look.